src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -14,7 +14,7 @@
 genericPrefixedOntology: Pattern[str] = regex.compile(r"^[\w\-]+:\w+$")
 
 
-def do_xml_consistency_check(onto_client: OntologyClient, root: etree._Element) -> None:
+def do_xml_consistency_check_with_ontology(onto_client: OntologyClient, root: etree._Element) -> None:
```

Comment:
@BalduinLandolt 
I am not sure how I want to re-organise this. For the TextValue check we need the same response fromt the API, but we need to filter out different information. So I do want to combine it, but I am unsure if it is a good idea to add the lower level functions that do the specific operations into this file. I think it could get very confusing as these checks do similar but slightly different things.
What do you think?

Reply:
not super familiar with this code... could "low level stuff" be hidden inside the OntologyClient? Otherwise maybe we could discuss the issue in detail together, so I understand a bit better

Reply:
That would be nice thanks!